packages/agent/conf/demos/knb/auckland/templates/auckland/NotaryCompanyVerification.hbs

```diff
@@ -8,7 +8,7 @@
     ],
     "expirationDate": "{{{ dateTimeAfterMonths 12 }}}",
     "credentialSubject": {
-        "Omschrijving": "Bedrijf is geverifieerd door de notaris",
+        "description": "Company has been verified by the notary",
```

Comment:
are we sure this should now be english?

Reply:
I reverted this along with Voornaam -> firstName
Actually this would be a correct i18n way:
```
{
    "@context": [
        "https://www.w3.org/2018/credentials/v1", {
            "description": {
                "@id": "http://schema.org/description",
                "@container": "@language"
            },
        }
    ],
    "type": [
        "VerifiableCredential",
        "NotaryCompanyVerification"
    ],
    "expirationDate": "{{{ dateTimeAfterMonths 12 }}}",
    "credentialSubject": {
        "description": {
          "en": "Company has been verified by the notary",
          "nl": "Bedrijf is geverifieerd door de notaris"
         }
        {{{mergeJSON @root}}}
    }
}```

Does our wallet support that? Do we just dump the credentialSubject 1:1 or do we actually read from it?